python/ray/air/examples/dreambooth/train.py

```diff
@@ -129,6 +129,10 @@ def train_fn(config):
 
     global_step = 0
     for epoch in range(num_train_epochs):
+        if global_step >= config["max_train_steps"]:
+            print(f"Stopping training after reaching {global_step} steps...")
```

Comment:
Is it duplicated? We can remove this and only keep the one below.

Reply:
Oh, I think break will only get out of the inner loop, so I need to break again.